src/transaction/atomic_replication_helper.cpp

```diff
@@ -57,20 +55,25 @@ namespace cublog
   }
 
 #if !defined (NDEBUG)
-  bool atomic_replication_helper::is_page_part_of_atomic_replication_sequence (TRANID tranid, VPID vpid) const
+  bool atomic_replication_helper::is_page_part_of_atomic_replication_sequence (VPID vpid) const
```

Comment:
- rename to `void check_log_record_vs_transaction`
- if vpid is not part of an atomic replication sequence, ok, don't care
- if it is part of an atomic replication sequence, its `tranid` must be the same.